drivers/pds/lib/utils.go

```diff
@@ -914,6 +914,30 @@ func CreatepostgresqlWorkload(dnsEndpoint string, pdsPassword string, scalefacto
 	return deployment, err
 }
 
+func CollectPodLogsandValidateWorkloads() error {
+	pods, err := k8sCore.GetPods("automation", nil)
+	if err != nil {
+		log.Errorf("An Error occured while getting pods")
```

Comment:
Why are we logging this error instead of returning?
`return fmt.Errorf("Failed to get pods, Err; %v", err)